apps/teams/helpers.py

```diff
@@ -57,6 +56,5 @@ def create_default_team_for_user(user: CustomUser, team_name: str = None):
     team_name = team_name or get_default_team_name_for_user(user)
     slug = get_next_unique_team_slug(team_name)
     team = Team.objects.create(name=team_name, slug=slug)
-    team.members.add(user, through_defaults={"role": roles.ROLE_ADMIN})
-    team.save()
+    make_user_team_owner(team, user)
```

Comment:
Not a biggy, but what are your thoughts on making the `make_user_team_owner` and creating the team more coupled? Maybe calling `make_user_team_owner` inside the save function after the super().save call or something. Up to you

Reply:
I kept it separate for the sake of tests since there are places where we create 2 'owners' for the same team.

I could create another utility which is used here that does both.

Reply:
Fair, thanks for clarifying